misk-clustering-dynamodb/src/test/kotlin/misk/clustering/DynamoClusterTest.kt

```diff
@@ -31,10 +31,10 @@ class DynamoClusterTest {
       install(FakeClusterWeightModule())
       install(
         InProcessDynamoDbModule(
-          DynamoDbTable("misk-cluster-members", DyClusterMember::class),
+          DynamoDbTable("$TEST_SERVICE_NAME.misk-cluster-members", DyClusterMember::class),
```

Comment:
Is this change backwards compatible? Doesn't this change the location in Dynamo where Misk will look for the table?

Reply:
It is, but it isn't being used in any repo currently.